hypervisor/arch/x86/rdt.c

```diff
@@ -22,7 +22,7 @@ const uint16_t hv_clos = 0U;
  * clos value (valid_clos_num) that is common between the resources as
  * each resource's clos max value to have consistent allocation.
  */
-uint16_t valid_clos_num = MAX_PLATFORM_CLOS_NUM;
```

Comment:
why change MAX_PLATFORM_CLOS_NUM to MAX_CACHE_CLOS_NUM_ENTRIES? These have 2 different meanings.
MAX_PLATFORM_CLOS_NUM is the common MAX CLOS supported among all RDT resources. But MAX_CACHE_CLOS_NUM_ENTRIES is the number of cache mask entries.  Here I believe it should be MAX_PLATFORM_CLOS_NUM.

Reply:
Ｉ have same question: Why we need this change? I didn't see the reasons.

Reply:
You are right, i have miss-understanding the 'MAX_PLATFORM_CLOS_NUM' before.
After sync with Vijay and Victor,  they agree with use MAX_COMMON_CLOS macro to instead MAX_PLATFORM_CLOS_NUM macro.

Reply:
If this is a simple rename, make it a separate patch and PR later with right explaination. Rename is less critical.

Reply:
@dongyaozu 
This is not only a rename. The previous understanding on MAX_PLATFORM_CLOS_NUM is not correct. We thought the value is decided by native board and it's scenario-irrelevant so put the MACRO in board_info.h.
But actually the value is presenting a common MAX CLOS supported among all RDT resources and this value depends on the setting of CONFIG_CDP_ENABLED, i.e. The MACRO should be scenario-relevant. So we rename the MACRO from MAX_PLATFORM_CLOS_NUM to MAX_COMMON_CLOS and put the definition into scenario specific misc_cfg.h.